### ISSUE TITLE
Rsync fail verification - data corruption or not written all

### ISSUE THREAD

**Issue by sergey-dryabzhinsky** [sergey-dryabzhinsky/dedupsqlfs#108]
```
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1668) [generator=3.1.2]
ERROR: files/usr/bin/mysqltest failed verification -- update retained.
ERROR: files/usr/bin/perror failed verification -- update retained.
ERROR: files/usr/bin/replace failed verification -- update retained.
ERROR: files/usr/lib/x86_64-linux-gnu/libicudata.so.52.1 failed verification -- update retained.
ERROR: files/usr/lib/x86_64-linux-gnu/libmysqlclient.so.18.0.0 failed verification -- update retained.
ERROR: files/usr/sbin/mysqld failed verification -- update retained.
ERROR: files/var/cache/apt/pkgcache.bin failed verification -- update retained.
ERROR: files/var/cache/apt/srcpkgcache.bin failed verification -- update retained.
ERROR: files/var/lib/mysql/rusoft_direct/direct_direct_phrase.MYD failed verification -- update retained.
ERROR: files/var/lib/mysql/rusoft_direct/direct_direct_statistic.MYD failed verification -- update retained.
ERROR: files/var/lib/mysql/rusoft_direct/direct_direct_statistic_price.MYD failed verification -- update retained.
```

Maybe not all blocks written.
Or again errors with LZ4 or other method?

**Comment by sergey-dryabzhinsky** [comment]
Seems fixed by: https://github.com/sergey-dryabzhinsky/dedupsqlfs/commit/15ba14a188debf406749f2e5a926391afbc51eed

Need to allocate block by it real size.
Need to store somewhere real block size.

**Comment by sergey-dryabzhinsky** [comment]
RealSize must be stored in `index` table.

**Comment by sergey-dryabzhinsky** [comment]
Fixed in #112
